src/plugins/kibana/public/discover/controllers/discover.js

```diff
@@ -136,6 +136,7 @@ define(function (require) {
       index: $scope.indexPattern.id,
       timefield: $scope.indexPattern.timeFieldName,
       savedSearch: savedSearch,
+      highlight: config.get('discover:highlight'),
```

Comment:
Attaching this to `$scope.opts` is not necessary, this value isn't used outside of the immediate context.